src/analytics/private/sqls/redshift/event-v2.sql

```diff
@@ -0,0 +1,99 @@
+CREATE TABLE IF NOT EXISTS {{schema}}.event_v2 (
+    event_timestamp timestamp not null,
+    event_id varchar(255) not null,
+    event_time_msec bigint not null,
+    event_name varchar(255) not null,
+    event_value double precision,
+    event_value_currency varchar(32),
+    event_bundle_sequence_id bigint,
+    ingest_time_msec bigint,
+    device_mobile_brand_name varchar(255),
+    device_mobile_model_name varchar(255),
+    device_manufacturer varchar(255),
+    device_carrier varchar(255),
+    device_network_type varchar(255),
+    device_operating_system varchar(255),
+    device_operating_system_version varchar(255),
+    device_vendor_id varchar(255),
+    device_advertising_id varchar(255),
+    device_system_language varchar(255),
+    device_time_zone_offset_seconds int,
+    device_ua_browser varchar(255),
+    device_ua_browser_version varchar(255),
+    device_ua_device varchar(255),
+    device_ua_device_category varchar(255),
+    device_screen_width int,
+    device_screen_height int,
+    device_viewport_width int,
+    device_viewport_height int,
+    geo_continent varchar(255),
+    geo_sub_continent varchar(255),
+    geo_country varchar(255),
+    geo_region varchar(255),
+    geo_metro varchar(255),
+    geo_city varchar(255),
+    geo_locale varchar(255),
+    traffic_source_source varchar(255),
+    traffic_source_medium varchar(255),
+    traffic_source_campaign varchar(255),
+    traffic_source_content varchar(2048),
+    traffic_source_term varchar(2048),
+    traffic_source_campaign_id varchar(255),
+    traffic_source_clid_platform varchar(255),
+    traffic_source_clid varchar(2048),
+    traffic_source_channel_group varchar(255),
+    user_first_touch_time_msec bigint,
```

Comment:
+ traffic_source_category (varchar(255) -- this will store the category value looked up from the the source_category table during traffic source processing, e.g., for google.com, this field will be 'search'

Reply:
added